docs/graphql/manual/api-reference/graphql-api/query.rst

```diff
@@ -453,13 +499,35 @@ NotExp
       _not: BoolExp_
     }
 
+*Example*
+
+.. code-block:: graphql
+
+  query {
+    article(where: {_not: {title: {_eq: ""}}} ) {
+      title
+      content
+    }
+  }
+
 TrueExp
 #######
 
 .. parsed-literal::
 
     {}
 
+*Example*
```

Comment:
Maybe a quick note on what the behaviour here would be will really help. The TrueExp is a little confusing to understand. Also the example here is not the best. The TrueExp is mostly useful to check if some objects exist and hence makes sense to use for nested objects. `is_published: {}` will always return `true` as it will always exist (even if it is null)